Documentation/CustomFormatters.md

```diff
@@ -154,41 +162,36 @@ MyCustomFormatter.m
 
 @implementation MyCustomFormatter
 
-- (id)init
-{
-    if((self = [super init]))
-    {
+- (id)init {
+    if((self = [super init])) {
         threadUnsafeDateFormatter = [[NSDateFormatter alloc] init];
         [threadUnsafeDateFormatter setDateFormat:@"yyyy/MM/dd HH:mm:ss:SSS"];
     }
     return self;
 }
 
-- (NSString *)formatLogMessage:(DDLogMessage *)logMessage
-{
+- (NSString *)formatLogMessage:(DDLogMessage *)logMessage {
     NSString *logLevel;
-    switch (logMessage->logFlag)
-    {
-        case LOG_FLAG_ERROR : logLevel = @"E"; break;
-        case LOG_FLAG_WARN  : logLevel = @"W"; break;
-        case LOG_FLAG_INFO  : logLevel = @"I"; break;
-        case LOG_FLAG_DEBUG : logLevel = @"D"; break;
+    switch (logMessage.flag) {
+        case DDLogFlagError : logLevel = @"E"; break;
+        case DDLogFlagWarn  : logLevel = @"W"; break;
+        case DDLogFlagInfo  : logLevel = @"I"; break;
+        case DDLogFlagDebug : logLevel = @"D"; break;
         default             : logLevel = @"V"; break;
     }
-    
-    NSString *dateAndTime = [threadUnsafeDateFormatter stringFromDate:(logMessage->timestamp)];
-    NSString *logMsg = logMessage->logMsg;
+
+    NSString *dateAndTime = [threadUnsafeDateFormatter stringFromDate:(logMessage.timestamp)];
+    NSString *logMsg = logMessage.message;
```

Comment:
I think that for performance reasons direct ivar access was preferred in heavily called methods like this one.